File: langgraph/data/semantic_equiv/121.py
def calculate_even_index_odd_values_sum(numbers):
    total = 0
    index = 0
    while index < len(numbers):
        if index % 2 == 0 and numbers[index] % 2 == 1:
            total += numbers[index]
        index += 1
    return total

# method name: solution -> calculate_even_index_odd_values_sum
# parameter name: lst -> numbers
# replaced list comprehension with a while loop
# introduced a total variable to accumulate the sum
# used a while loop with an index variable instead of enumerate

File: langgraph/result/qwen/14b-eq2js/121.js
function calculateOddSumAtEvenIndices(numbers) {
    let total = 0;
    let index = 0;
    while (index < numbers.length) {
        if (index % 2 === 0 && numbers[index] % 2 === 1) {
            total += numbers[index];
        }
        index += 1;
    }
    return total;
}

// method name: solution -> calculateOddSumAtEvenIndices
// parameter name: lst -> numbers
// replaced list comprehension with while loop and if condition
// used a manual sum calculation instead of sum() on a list comprehension